docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,237 @@
+
+### The `stash:///` URL Scheme:
+This URL scheme comes from legacy code called `stashcp` that Pelican was built off of and is still accepted for backwards compatibility. This URL scheme functions identically to the `osdf:///` URL scheme, loading in OSDF defaults. Here is an example:
+```bash
+pelican object get stash:///<namespace-prefix></path/to/file> <local/path/to/file>
+```
+
+## Get a Public Object from your Federation
+
+To use the pelican client to get public objects from a federation, use Pelican's `object get` sub-command
+
+### Getting an Object Using the pelican:// URL Scheme:
+To use the `pelican://` URL scheme, you need to specify the federation URL within the request URL. To do this, you would format the `object get` command like so:
+```bash
+pelican object get pelican://<federation-url></namespace-prefix></path/to/file> <local/path/to/file>
+```
+Or, you can use the `-f` flag with the command, and format the request URL like so:
+```bash
+pelican object get -f <federation-url> pelican://</namespace-prefix></path/to/file> </local/path/to/file>
+```
+You can try this yourself by getting the public file that was mentioned earlier from the OSDF. Using the `object get` sub command, and indicating to pelican that you're pulling from the OSDF:
+
+```bash
+pelican object get pelican://osg-htc.org/ospool/uc-shared/public/OSG-Staff/validation/test.txt downloaded-testfile.txt
+```
+Or:
+```bash
+pelican object get -f https://osg-htc.org pelican:///ospool/uc-shared/public/OSG-Staff/validation/test.txt downloaded-testfile.txt
+```
+
+This command will download the object `/OSG-Staff/validation/test.txt` from the OSDF's `/ospool/uc-shared/public/` namespace and save it in your local directory with the name `downloaded-testfile.txt`.
+
+More specifically, it breaks into these components:
+- Federation URL: `osg-htc.org`
+- Namespace: `/ospool/uc-shared/public`
+- Object name: `/OSG-Staff/validation/test.txt`
+- Destination: `downloaded-testfile.txt`
+
+
+## Get A Protected Object From Your Federation
+
+Protected namespaces require that a Pelican client prove it is allowed to access objects from the namespace before the object can be downloaded. In many cases, Pelican clients can do this automatically by initiating an OpenID-Connect (OIDC) flow that uses an external log-in service (such as [CILogon](https://www.cilogon.org)) through your browser. In other cases, a token must be provided to the Pelican client manually.
+
+### For Issuers That Support CILogon Code Flow
+
+Some origins support authentication with CILogon's OIDC client. In these cases, the Pelican client is capable of handling authentication with the origin automatically, once you logged in via CILogon. To download protected objects from origins that support CILogon, run the same command as for downloading a public object:
```

Comment:
Same comments about tidying up the language around the "automatically" phrasing.